lms/djangoapps/teams/static/teams/js/spec_helpers/team_spec_helpers.js

```diff
@@ -228,6 +230,38 @@ define([
         );
     };
 
+    createMockTopicData = function (startIndex, stopIndex) {
+        return _.map(_.range(startIndex, stopIndex + 1), function (i) {
+            return {
+                "description": "description " + i,
+                "name": "topic " + i,
+                "id": "id " + i,
+                "team_count": 0
+            };
+        });
+    };
+
+    createMockTopicCollection = function (topicData) {
+        topicData = topicData !== undefined ? topicData : createMockTopicData(1, 5);
+
+        return new TopicCollection(
+            {
+                count: topicData.length + 1,
+                current_page: 1,
+                num_pages: 2,
```

Comment:
It is necessary, and it exists in "real" JSON from the server. I was wrong to remove it in the other place-- it wasn't being exercised there, but it really should be present (to clarify, I had removed num_pages in the earlier PR).